firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -73,6 +78,11 @@
   /* file used for process-level syncronization of generating and persisting fids */
   private static final String LOCKFILE_NAME_GENERATE_FID = "generatefid.lock";
 
+  private static final String IID_SDK_PROP_FILE = "/firebase-iid.properties";
```

Comment:
Please add a code comment why you need this.
Something like
// Values that are needed to prevent incompatible versions of Instance-ID SDK to be used in parallel